### DISCUSSION TITLE
Is the minimum OBS 30 version really necessary?

### DISCUSSION THREAD

**Discussion by themegaxandy** [WarmUpTill/SceneSwitcher#1190] [General]
I have an old laptop that I use to stream live, and the latest version of OBS that supports my processor is version 29. I love the plugin and using automations on it. It really changes the way my stream looks and in terms of quality. But since version 30 of OBS, QuickSync simply doesn't work anymore. Although the option still appears, it gives an error when trying to start a recording.

I understand if this is a permanent decision, and version 1.26.4 of the plugin already has a lot of features. So it's not all bad and it's okay if that's the case.

**Comment by WarmUpTill** [comment]
To stay compatible with the most recent OBS releases I will have to drop support for older OBS versions eventually.
The reason being that as OBS is evolving so is its API and eventually outdated functionality only available in older OBS releases will get dropped.

So unfortunately I don't really have much of a choice and the next OBS major release will even result in me having to bump the minimum supported version to 30.1.2.

If at some point you should urgently need a custom build of a newer version of the plugin compatible with OBS 29 don't hesitate to ask though. :)

**Comment by themegaxandy** [comment]
I completely understand. At some point, software has to stop supporting older versions, otherwise it won't evolve. And like I said, it's not all bad. Someday I'll upgrade my hardware as soon as I have the money.
